custom/icds_reports/sqldata/exports/beneficiary.py

```diff
@@ -84,7 +84,7 @@ def test_fucntion(x):
             return "%.2f" % x if x else "Data Not Entered"
 
         def phone_number_fucntion(x):
-            return '"{}"'.format(x) if x else x
+            return "'{}".format(x) if x else x
```

Comment:
why is there one `'` inside the double quotes. wont this look like `'911234567890`

Reply:
It is number formatting method, you can try it in every spreadsheet program, just write long enough number that it would be formatted in mathematical method eg `1234567890` and in another cell `'1234567890`. Apostrophe won't be presented.